docs/reference/high-availability/cluster-design.asciidoc

```diff
@@ -27,13 +24,18 @@ A resilient cluster needs three master-eligible nodes so that if one of
 them fails then the remaining two still form a majority and can hold a
 successful election.
 
-Similarly, node redundancy makes it likely that if a node for a particular role
-fails, another node can take on its responsibilities.
+Similarly, redundancy of nodes of each role means that if a node for a
+particular role fails, another node can take on its responsibilities.
 
 Finally, a resilient cluster should have at least two copies of each shard. If
-one copy fails then there is another good copy to take over. {es} automatically
-rebuilds any failed shard copies on the remaining nodes in order to restore the
-cluster to full health after a failure.
+one copy fails then there should be another good copy to take over. {es}
+automatically rebuilds any failed shard copies on the remaining nodes in order
+to restore the cluster to full health after a failure.
+
+Failures temporarily reduce the total capacity of your cluster. In addition,
```

Comment:
We mention this in the "larger clusters" section but it applies to all clusters so I thought it'd help to note it here too.